models/test/trigger_scale_legacy.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(
         schema='test',
-        alias = alias('trigger_scale_legacy'),
+        alias = alias('trigger_scale_legacy', legacy_model=True),
         tags= ['legacy','prod_exclude']
```

Comment:
Considering it has the prod_exclude tag, is this enough for not building this model on spark? (and therefore, if that's the case, then there wouldn't be a need to remove legacy from the model name)

Reply:
it is, but then again, that could cause issues if `test` schema doesn't exist on spark 🤔 i will monitor to see how it goes